active/NumberBox/NumberBox.md

```diff
@@ -0,0 +1,220 @@
+    Header="Enter equation:" 
+    PlaceholderText="000.000.000.000"
+    FormatType="IPAddress" />
+```
+
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for international telephone" 
+    Header="Enter equation:" 
+    PlaceholderText="0 (123) 456-7890"
+    FormatType="InternationalTelephone" />
+```
+
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for custom input" 
+    Header="Enter equation:" 
+    PlaceholderText="00 ft 00 in"
+    CustomFormatString="## ft ## in"
+    StripLeadingZeros="False" />
+```
+
+### Add increment and decrement stepping
+
+XAML
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for items in order" 
+    Header="Order:" 
+    Text="2"
+    StepSize="2"
+    UpDownArrowsEnabled="True"
+    UpDownDragEnabled="True" />
+```
+
+### Add a prefix and suffix 
+
+XAML
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for IP address" 
+    Header="Enter equation:" 
+    PlaceholderText="000.000.000.000" 
+    Prefix="IP Address:" />
+```
+
+XAML
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for cents" 
+    Header="Enter equation:" 
+    PlaceholderText="0.00" 
+    Suffix="cents" />
+```
+
+### Add mininum and maximum values
+
+XAML
+```XAML
+<TextBlock AutomationProperties.Name="NumberBox for items in order" 
+    Header="Order:" 
+    PlaceholderText="1" 
+    MinValue="1"
+    MaxValue="99" />
+```
+
+## Remarks
+<!-- Explanation and guidance that doesn't fit into the Examples section. -->
+
+<!-- APIs should only throw exceptions in exceptional conditions; basically,
+only when there's a bug in the caller, such as argument exception.  But if for some
+reason it's necessary for a caller to catch an exception from an API, call that
+out with an explanation either here or in the Examples -->
+
+## API Notes
+<!-- Option 1: Give a one or two line description of each API (type
+and member), or at least the ones that aren't obvious
+from their name.  These descriptions are what show up
+in IntelliSense. For properties, specify the default value of the property if it
+isn't the type's default (for example an int-typed property that doesn't default to zero.) -->
+
+<!-- Option 2: Put these descriptions in the below API Details section,
+with a "///" comment above the member or type. -->
+
+## API Details
+<!-- The exact API, in MIDL3 format (https://docs.microsoft.com/en-us/uwp/midl-3/) -->
+
+```c++ 
+enum NumberBoxFormatType
+{
+    IPAddress,
+    InternationalTelephone,
+    Currency,
+};
+
+unsealed runtimeclass NumberBox : Windows.UI.Xaml.Controls.TextBox
+{
+    NumberBox();
+    
+    String Prefix;
+    String Suffix;
+    
+    Boolean AcceptsCalculation;
+    Boolean UpDownArrowsEnabled;
+    Boolean UpDownDragEnabled;
+  
+    NumberBoxFormatType FormatType;
+    String CustomFormatString;
+    Boolean StripLeadingZeros;
+   
+    Integer MinValue;
+    Integer MaxValue;
+
+    static Windows.UI.Xaml.DependencyProperty PrefixProperty{ get; };
```

Comment:
How will the `Prefix` and `Suffix` be styled?
Alignment, spacing, foreground and background colors, and more need to be considered and the limits of customization also need to be specified.
Ideally, I'd like to see individual data templates be exposed for both these properties so they can be customized in the same way as `Header` without needing to retemplate the whole control. 
Without being able to see the intended designs for these properties it's hard to say how useful they would be by default.

Reply:
Additionally, and for confirmation, I'm assuming that in LTR languages, the prefix is displayed on the left and the suffix on the right. For RTL languages this will be reversed.

Reply:
This feedback should definitely be noted on the proposal-to-be for adding this feature improvement to NumberBox's parent class, TextBox. I will resolve this issue for this forum since prefix/suffix have been scoped out here. 

I wanted to make sure one of the original idea initiators had time to add that proposal, but I will come through and add it next week if I do not see it before then. Thanks for the always excellent input, @mrlacey!